l5kit/l5kit/geometry/angle.py

```diff
@@ -31,17 +31,16 @@ def compute_yaw_around_north_from_direction(direction_vector: np.ndarray) -> flo
     return angle_between_vectors(direction_vector, np.array([0.0, 1.0]))
 
 
-def angular_distance(angle_a: float, angle_b: float) -> float:
-    """
-    Return the angular distance (angle_a - angle_b) between two angles in radians.
-    The results is always in the [-pi, pi] range
+def angular_distance(angle_a: np.ndarray, angle_b: np.ndarray) -> np.ndarray:
```

Comment:
shouldn't this be a `Union(float, np.ndarray) `then ?

Reply:
@lucabergamini Done